tools/java/org/unicode/cldr/test/CheckUnits.java

```diff
@@ -93,8 +94,21 @@ public CheckCLDR handleCheck(String path, String fullPath, String value, Options
             }
             String idType;
             switch(pathType) {
+            case power: {
```

Comment:
This is slightly different than the ticket; cause an error but note to users that they can use gendered variants, eg feminine of 'cubic'